src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Don't declare `BASE_URL` in every file, just import it from `api.ts`.
```suggestion
import { BASE_URL } from './api';
```